swap_meet/vendor.py

```diff
@@ -1,2 +1,69 @@
+
 class Vendor:
-    pass
\ No newline at end of file
+    
+    def __init__(self, inventory=None):
+        self.inventory = [] if inventory is None else inventory
+
+
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else: 
+            return False
+
+
+    def add(self, element):
+        self.inventory.append(element)
+        return element
```

Comment:
Let's make the parameter a bit more descriptive, like `item`, so that other coders understand quickly what it is the function is doing.